frameworks/hdfs/src/main/dist/hdfs-site.xml

```diff
@@ -853,93 +841,102 @@
             <value>true</value>
         </property>
         <property>
-            <name>dfs.namenode.kerberos.principal.pattern</name>
-            <value>{{KERBEROS_PRIMARY}}/*@{{KERBEROS_REALM}}</value>
+            <name>dfs.block.access.key.update.interval</name>
+            <value>{{BLOCK_ACCESS_KEY_UPDATE_INTERVAL}}</value>
         </property>
         <property>
-            <name>dfs.datanode.kerberos.principal.pattern</name>
-            <value>{{KERBEROS_PRIMARY}}/*@{{KERBEROS_REALM}}</value>
-        </property>
-        <property>
-            <name>dfs.journalnode.kerberos.principal.pattern</name>
-            <value>{{KERBEROS_PRIMARY}}/*@{{KERBEROS_REALM}}</value>
+            <name>dfs.block.access.token.lifetime</name>
+            <value>{{BLOCK_ACCESS_TOKEN_LIFETIME}}</value>
         </property>
         <property>
             <name>dfs.cluster.administrators</name>
             <value>core,root,hdfs,nobody</value>
         </property>
         <property>
-            <name>dfs.web.authentication.kerberos.principal</name>
-            <value>{{KERBEROS_PRIMARY}}/{{TASK_NAME}}.{{FRAMEWORK_NAME}}.mesos@{{KERBEROS_REALM}}</value>
+            <name>dfs.journalnode.kerberos.principal.pattern</name>
+            <value>*</value>
         </property>
         <property>
-            <name>dfs.web.authentication.kerberos.keytab</name>
-            <value>keytabs/{{KERBEROS_PRIMARY}}.{{TASK_NAME}}.{{FRAMEWORK_NAME}}.mesos.keytab</value>
+            <name>dfs.namenode.kerberos.principal.pattern</name>
+            <value>*</value>
         </property>
-
-
-        {{#DATANODE}}
-        <!-- DataNode Security Configuration -->
         <property>
-            <name>dfs.datanode.keytab.file</name>
-            <value>keytabs/{{KERBEROS_PRIMARY}}.data-{{POD_INSTANCE_INDEX}}-node.{{FRAMEWORK_NAME}}.mesos.keytab</value>
+            <name>dfs.datanode.kerberos.principal.pattern</name>
+            <value>*</value>
         </property>
         <property>
-            <name>dfs.datanode.kerberos.principal</name>
-            <value>{{KERBEROS_PRIMARY}}/data-{{POD_INSTANCE_INDEX}}-node.{{FRAMEWORK_NAME}}.mesos@{{KERBEROS_REALM}}</value>
+            <name>dfs.web.authentication.kerberos.principal</name>
+            <value>*</value>
         </property>
         <property>
-            <name>dfs.datanode.data.dir.perm</name>
```

Comment:
Don't we need the datanode.data.dir.perm property?

Reply:
Yeah, it was just moved.